### PR TITLE
Update for the version 4.10 release

### PR DESCRIPTION
**Pull request by jrich523**
Updated logic to handle 4.10+

Testing

- installed via 4.9 CP and sidecar
- upgraded 4.9 to 4.10
- clean install with 4.10
- clean install with no version

### CODE REVIEW

File: ADVANCED.md [hunk 1]
```diff
@@ -1,13 +1,35 @@
 # Advanced configuration
 
+Most of these configuration options will leverage the env file set by `ENV_FILE_PATH` to
+provide these variables to the sidecar.
+
+## Database Accounts
+
+If you are using local credentails for testing, as part of the install script you will have the option to use an enviroment file to provide those credentials to the sidecar.
```
Comment (wcmjunior) [inline ADVANCED.md]:
```suggestion
If you are using local credentials for testing, as part of the install script you will have the option to use an environment file to provide those credentials to the sidecar.
```

File: ADVANCED.md [hunk 2]
```diff
@@ -1,13 +1,35 @@
...
+
+The value of the secret should be in the following format
```
Comment (wcmjunior) [inline ADVANCED.md]: `should` or `must`?

File: README.md [hunk 3]
```diff
@@ -1,12 +1,8 @@
...
+# Sidecar - Express Install
 
 A quick start for a Express to get a simple sidecar online quickly!
 
-Install with a single command
-
-```sh
-bash -c "$(curl -fsSL https://raw.githubusercontent.com/cyral-quickstart/quickstart-sidecar-express/main/install-sidecar.sh)"
-```
+An install command can be generated on the sidecar deployment page.
```
Comment (wcmjunior) [inline README.md:5]:
```suggestion
An install command can be generated on the sidecar deployment page.

Learn more in the [Sidecar Deployment](https://cyral.com/docs/sidecars/deployment/) page.
```

**Review by gabfelp — COMMENTED** [review]
I made a test using v4.10.0 and the script is working fine:
image
image


Also notice that the default behavior of the script when the sidecar version is not provided is:
- Request version to the CP 

And this worked fine too -- was able to get the correct version


Besides that, I don't see shellcheck complaining about anything so I'm good with the script.

File: ADVANCED.md [hunk 4]
```diff
@@ -1,13 +1,35 @@
 # Advanced configuration
 
+Most of these configuration options will leverage the env file set by `ENV_FILE_PATH` to
+provide these variables to the sidecar.
+
+## Database Accounts
+
+If you are using local credentials for testing, as part of the install script you will have the option to use an environment file to provide those credentials to the sidecar.
+
+The value of the secret should be in the following format
```
Comment (wcmjunior) [inline ADVANCED.md]:
```suggestion
The value of the secret must be in the following format
```

**Review by wcmjunior — APPROVED** [review]

**Pull request merged**